influxql/engine.go

```diff
@@ -930,6 +932,16 @@ func (e *Executor) execute(out chan *Row) {
 	close(out)
 }
 
+func fromInt64ToFloat64(v interface{}) float64 {
```

Comment:
Can you rename this to `i64tof64?  We usually keep their names succinct like: https://github.com/influxdb/influxdb/blob/master/tsdb/shard.go#L806